configuration/pih/htmlforms/drugRehab.xml

```diff
@@ -214,11 +218,69 @@
         </includeIf>
     </ifMode>
 
-    <section id="consult" sectionTag="fieldset" headerTag="legend" headerStyle="title" headerCode="consult">
-        <div class="section-container">
-            <label>
-                <uimessage code="add form here" />
-            </label>
+    <section id="mh_scores" sectionTag="fieldset" headerTag="legend" headerStyle="title" headerCode="pihcore.drugRehab">
```

Comment:
Change the id.  "mh_scores" is old.  Otherwise it looks good @Dutervil

Reply:
I can change it to anything ? @lnball

Reply:
it anything it fine, you can merge for testing @lnball

Reply:
I normally change the id to what's in the section.  You call the section "Drug Rehab"?   Will there only be 1 section on the form?

Reply:
One Section for now